### PR TITLE
Made Changes to How to use SwiftCurrent with UIKit Guide

### PR DESCRIPTION
**Pull request by courtneybeisner**
Proofread and made 3 changes.

## Checklist:
- [ ] Did you sign the [Contributor License Agreement](https://cla-assistant.io/wwt/SwiftCurrent)?
- [ ] Is the linter reporting 0 errors?
- [ ] Did you comply with our [styleguide](https://github.com/wwt/SwiftCurrent/blob/main/.github/STYLEGUIDE.md)?
- [ ] Is there [adequate test coverage](https://github.com/wwt/SwiftCurrent/blob/main/.github/CONTRIBUTING.md#test-etiquette) for your new code?
- [ ] Does the CI pipeline pass?
- [ ] Did you [update the documentation](https://github.com/wwt/SwiftCurrent/blob/main/.github/CONTRIBUTING.md#documentation)?
- [ ] Did you [update the sample app](https://github.com/wwt/SwiftCurrent/blob/main/.github/CONTRIBUTING.md#sample-app)?
- [ ] Do we need to [increment the minor/major version](https://github.com/wwt/SwiftCurrent/blob/main/.github/CONTRIBUTING.md#versioning)?
- [ ] Did you [change the public API](https://github.com/wwt/SwiftCurrent/blob/main/.github/CONTRIBUTING.md#public-api)?
- [ ] Have you done everything you can to make sure that errors that can occur are compile-time errors, and if they have to be runtime do you have adequate tests and documentation around those runtime errors? [For more details](https://github.com/wwt/SwiftCurrent/blob/main/.github/CONTRIBUTING.md#errors).

----

### If Applicable:
- [ ] Did you test when the first item is skipped?
- [ ] Did you test when the last item is skipped?
- [ ] Did you test when middle items are skipped?
- [ ] Did you test when incorrect data is passed forward?
- [ ] Did you test proceeding backwards?

----

### If Public API Has Changed:
- [ ] Did you deprecate (rather than remove) any old methods/variables/etc? [Our philosophy for deprecation](https://github.com/wwt/SwiftCurrent/blob/main/.github/CONTRIBUTING.md#deprecation).
- [ ] Have you done the best that you can to make sure that the compiler guides people to changing to the new API? (Example: the renamed attribute)
- [ ] If necessary, have you tested the upgrade path for at least N-1 versions? For example, if data persists between v1 and v2 then that upgrade should be tested and as easy as we can make it.

### CODE REVIEW

**Review by Richard-Gist — DISMISSED** [review]

File: .github/abstract/How to use SwiftCurrent with UIKit.md [hunk 1]
```diff
@@ -1 +1 @@
-SwiftCurrent works well with UIKit. To work correctly it does require `FlowRepresentable`s to be `UIViewController`s and not `UIView`s. It comes with [support for Storyboards](using-storyboards.html) with `StoryboardLoadable` and [programmatic views](using-programmatic-views.html). 
\ No newline at end of file
+SwiftCurrent works well with UIKit. To work correctly, requires `FlowRepresentable`s to be `UIViewController`s and not `UIView`s. It comes with [support for Storyboards](using-storyboards.html) with `StoryboardLoadable` and [programmatic views](using-programmatic-views.html).
```
Comment (Tyler-Keith-Thompson) [inline .github/abstract/How to use SwiftCurrent with UIKit.md:1]: Meaning got lost here I think???

I had to read this several times to understand the change. 

Verbose version:

To work correctly you done gotta make your `FlowRepresentable`s `UIViewController`s, if you make them `UIViews` things'll break, so don't do that.

Comment (Tyler-Keith-Thompson) [reply .github/abstract/How to use SwiftCurrent with UIKit.md:1]:
```suggestion
SwiftCurrent works well with UIKit. To work correctly, `FlowRepresentable`s need to be `UIViewController`s and not `UIView`s. It comes with [support for Storyboards](using-storyboards.html) with `StoryboardLoadable` and [programmatic views](using-programmatic-views.html). 
```

Comment (Richard-Gist) [reply .github/abstract/How to use SwiftCurrent with UIKit.md:1]: what about? 
```suggestion
SwiftCurrent works well with UIKit when `FlowRepresentable`s are `UIViewController`s. `UIView`s are not supported at this time. It comes with [support for Storyboards](using-storyboards.html) with `StoryboardLoadable` and [programmatic views](using-programmatic-views.html). 
```

Comment (morganzellers) [reply .github/abstract/How to use SwiftCurrent with UIKit.md:1]: +1

Comment (courtneybeisner) [reply .github/abstract/How to use SwiftCurrent with UIKit.md:1]: @Richard-Gist Sounds good to me!

**Review by morganzellers — APPROVED** [review]

**Pull request merged**